Nightlies not publishing

```
[07:04:48] Starting 'buildServerLibraryOut'...
##[error]src/harness/tsconfig.json(4,9): error TS1391: The `bundledPackageName` option must be provided when using outFile and node module resolution with declaration emit.
```

https://github.com/microsoft/TypeScript/runs/1105196341?check_suite_focus=true#step:4:132

Seems broken by https://github.com/microsoft/TypeScript/pull/39277

--- a/src/harness/tsconfig.json
+++ b/src/harness/tsconfig.json
@@ -2,7 +2,6 @@
     "extends": "../tsconfig-base",
     "compilerOptions": {
         "outFile": "../../built/local/harness.js",
-        "moduleResolution": "node",
         "types": [
             "node", "mocha", "chai"
         ],

